features/map.yml

```diff
@@ -18,33 +18,5 @@ spec:
 snapshot: ecmascript-2015
 group: maps
 status:
-  baseline: high
-  baseline_low_date: 2015-07-29
-  # Editorial override has been applied to ignore Map.@@species
-  # https://github.com/web-platform-dx/web-features/pull/712#discussion_r1532483885
-  support:
-    chrome: "38"
-    chrome_android: "38"
-    edge: "12"
-    firefox: "13"
-    firefox_android: "14"
-    safari: "8"
-    safari_ios: "8"
-compat_features:
-  - javascript.builtins.Map
-  - javascript.builtins.Map.Map
-  - javascript.builtins.Map.Map.iterable_allowed
-  - javascript.builtins.Map.Map.null_allowed
-  - javascript.builtins.Map.clear
-  - javascript.builtins.Map.delete
-  - javascript.builtins.Map.entries
-  - javascript.builtins.Map.forEach
-  - javascript.builtins.Map.get
-  - javascript.builtins.Map.has
-  - javascript.builtins.Map.key_equality_for_zeros
-  - javascript.builtins.Map.keys
-  - javascript.builtins.Map.set
-  - javascript.builtins.Map.size
-  - javascript.builtins.Map.values
-  - javascript.builtins.Map.@@iterator
-  - javascript.builtins.Map.@@species
+  compute_from:
+    - javascript.builtins.Map.Map
```

Comment:
I probably don't have the full context here, but wouldn't something like this be a bit more intuitive:

```yaml
status:
  compat_features:
    - javascript.builtins.Map.*
```

Pros: no need for a new field in the YAML (`compute_from`), and the `*` character is well known to developers.

But, again, maybe this is not reflecting fully the idea of the feature here.

Reply:
I tried to add more context with https://github.com/web-platform-dx/web-features/pull/1227#issuecomment-2160997849.

To the specific wildcard suggestion, I think it's contrary to the idea of pinning to specific keys. I also worry that it would be confusing, since it would be a wildcard against the keys in `compat_features` (or tagged in BCD), not all keys matching the pattern.